src/client.js

```diff
@@ -629,6 +629,7 @@ const fileUploader = (client) => {
                                     size: file.size,
                                 };
                                 if (action === "publishIfNeeded") {
+                                    try{
```

Comment:
It's a small thing, but could you add spaces before the { to remain consistent with the rest of the code, please.